.github/workflows/publish-commit-bottles.yml

```
@@ -26,6 +26,26 @@ jobs:
             console.log("Dispatched by " + user.data.name + " <" + email + ">")
             core.setOutput("name", user.data.name)
             core.setOutput("email", email)
+      - name: Post comment once started
+        uses: actions/github-script@0.9.0
+        with:
+          github-token: ${{secrets.HOMEBREW_GITHUB_API_TOKEN}}
+          script: |
+            const run_id = process.env.GITHUB_RUN_ID
+            const actor = context.actor
+            const pr = context.payload.client_payload.pull_request
+            console.log("run_id=" + run_id)
+            console.log("actor=" + actor)
+            console.log("pr=" + pr)
+
+            const repository = context.repo.owner + '/' + context.repo.repo
+            const url = 'https://github.com/' + repository + '/actions/runs/' + run_id
+
+            github.issues.createComment({
+              ...context.repo,
+              issue_number: pr,
+              body: '@' + actor + ' has triggered a bottle pull:' + url
```

This might make it clearer to new contributors what's going on.
```suggestion
              body: '@' + actor + ' has triggered a merge, running bottle pull:' + url
```

---

Thanks, that's a good idea.